packages/components/pie-checkbox/src/checkbox.scss

```diff
@@ -1 +1,276 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+@keyframes checkboxCheck {
+    0% {
+        width: 0;
+        height: 0;
+        border-color: #fff;
+
+        transform: translate3d(0, 0, 0) rotate(45deg);
+    }
+
+    33% {
+        width: 8px;
+        height: 0;
+
+        transform: translate3d(0, 0, 0) rotate(45deg);
+    }
+
+    100% {
+        width: 8px;
+        height: 16px;
+        border-color: #fff;
+        border-bottom-right-radius: 2px;
+
+        transform: translate3d(0, -16px, 0) rotate(45deg);
+    }
+}
```

Comment:
I appreciate I'm not a designer but I find this animation very distracting. Is this something they have requested? Entirely personal opinion but I think this needs to be toned down quite a lot

Reply:
This is what was approved for snacks when the motion work was initiated. But I will definitely request design review after the engineering one.

Reply:
Animation timing was updated slightly and Molly approved the result